components/mpas-ocean/src/shared/mpas_ocn_surface_land_ice_fluxes.F

```diff
@@ -650,6 +655,17 @@ subroutine ocn_surface_land_ice_fluxes_build_arrays(meshPool, &
 
       endif ! jenkinsOn or hollandJenkinsOn
 
+      ! Add frazil and interface melt/freeze to get total fresh water flux
+      if ( associated(frazilIceFreshwaterFlux) ) then
+          do iCell = 1, nCells
```

Comment:
```suggestion
          do iCell = 1, nCells
             if (landIceFloatingMask(iCell) == 0) cycle
```
You might consider including this line as above to save computations. I haven't done a performance comparison though to see if it actually saves time if you only have a few FLOPs per cell.

Reply:
I think that wouldn't make much difference in the grand scheme of things because most cores would just wait for the few that own land ice to finish.  But it doesn't hurt...